internal/proxy/proxy.go

```diff
@@ -461,8 +491,22 @@ func (p *Proxy) processToken(tokenData []byte) (pci ProxyCacheItem, osioToken st
 }
 
 //HandleError creates a JSON response with a given error and writes it to ResponseWriter
-func (p *Proxy) HandleError(w http.ResponseWriter, err error) {
-	log.Error(err)
+func (p *Proxy) HandleError(w http.ResponseWriter, err error, requestLogEntry *log.Entry) {
+	// log the error
+	location := ""
+	if err != nil {
+		pc, fn, line, _ := runtime.Caller(1)
+
+		location = fmt.Sprintf(" %s[%s:%d]", runtime.FuncForPC(pc).Name(), fn, line)
```

Comment:
This is awesome:)

Reply:
thanks :-) Definitely adds some more context.